src/components/modules/HistorySection/HistorySection.tsx

```diff
@@ -0,0 +1,90 @@
+import { useCallback, useEffect, useState } from 'react'
+
+import { NextImage } from '@/elements/NextImage/NextImage'
+
+export const HistorySection = () => {
+  const [active, setActive] = useState(0)
+  const [next, setNext] = useState(1)
+  const [prev, setPrev] = useState(2)
+  const [activeStyle, setActiveStyle] = useState('z-10')
+  const [nextStyle, setNextStyle] = useState('translate-x-[200px] translate-y-[50px]')
+  const [prevStyle, setPrevStyle] = useState('translate-x-[-200px] translate-y-[50px]')
+
+  const COMPETITIONS = [
+    {
+      title: 'Lomba 1',
+      organizer: 'Universitas Indonesia',
+      img: 'https://cdn.discordapp.com/attachments/826088171212636211/959083610831474729/compfest13-color_1.jpg',
+    },
+    {
+      title: 'Lomba 2',
+      organizer: 'Universitas Gajah Mada',
+      img: 'https://cdn.discordapp.com/attachments/826088171212636211/959083610831474729/compfest13-color_1.jpg',
+    },
+    {
+      title: 'Lomba 3',
+      organizer: 'Universitas Brawijaya',
+      img: 'https://cdn.discordapp.com/attachments/826088171212636211/959083610831474729/compfest13-color_1.jpg',
+    },
+  ]
+
+  const handleNextSlide = useCallback(() => {
+    setActiveStyle('translate-x-[-200px] translate-y-[50px]')
+    setNextStyle('z-10')
+    setPrevStyle('translate-x-[200px] translate-y-[50px]')
```

Comment:
ini gabisa pakai conditional di classnamenya aja han? jadi statenya cuman perlu isActive, setIsActive

Reply:
nanti buat ngatur delaynya gimana ka?

Reply:
kalo gini gimana?

```
const handleNextSlide = useCallback(() => {
  setActive({ index: active.index, style: 'translate-x-[-200px] translate-y-[50px]' })
  setNext({ index: next.index, style: 'z-10' })
  setPrev({ index: prev.index, style: 'translate-x-[200px] translate-y-[50px]' })
  setTimeout(() => {
    setActive({ index: (active.index + 1) % 3, style: 'z-10' })
    setNext({ index: (next.index + 1) % 3, style: 'translate-x-[200px] translate-y-[50px]' })
    setPrev({ index: (prev.index + 1) % 3, style: 'translate-x-[-200px] translate-y-[50px]' })
  }, 300)
}, [active, next, prev])
```

Reply:
delaynya di setTimeout itu kan? mksd gw stylenya itu biar gadi taro di state langsung di classNamenya. posisi historynya diketahuin dari isActive  
 className={`absolute flex h-[300px] w-[300px] rounded-[20px] bg-white transition duration-500 ease-in-out ${
              index == active ? style aktif disini 
            } ${ :index == next ? next style disini }
            ${ :index == prev ? prev style disini }
            `}